flink-elastic-catalog/src/main/java/com/getindata/flink/connector/jdbc/catalog/ElasticCatalog.java

```diff
@@ -453,10 +489,49 @@ public void setScanPartitionUpperBound(Long upperBound) {
         }
 
         public String toString() {
-            return "partitionColumnName=" + partitionColumnName +
-                    ", partitionNumber=" + partitionNumber +
-                    ", scanPartitionLowerBound=" + scanPartitionLowerBound +
-                    ", scanPartitionUpperBound=" + scanPartitionUpperBound;
+            return "partitionColumnName=" + partitionColumnName + ", partitionNumber=" + partitionNumber + ", scanPartitionLowerBound=" + scanPartitionLowerBound + ", scanPartitionUpperBound=" + scanPartitionUpperBound;
+        }
+    }
+
+    static class TimeAttributeProperties {
+        private String watermarkColumn;
+        private String watermarkDelay;
+        private String proctimeColumn;
+
+        public TimeAttributeProperties() {
```

Comment:
You dont need to define default constructor which sets all fields to null. If no constructor is defined, the default one is created.